.changeset/perky-snails-feel.md

```
@@ -0,0 +1,15 @@
+---
+'@mastra/core': patch
+---
+
+Fixed structured output option typing so processor-only fields require a separate `structuredOutput.model`.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)